docs/howto/redis_cluster.md

```diff
@@ -0,0 +1,41 @@
+# Redis Cluster
+
+commcare-cloud supports setting up Redis as a monolith or as a cluster. Setting up monolith is simple and is in built in the full stack deploy. Simply add the host to `redis` host group in the inventory. Setting up a cluster is not fully automated in commcare-cloud and there are few additional steps required. This document explains how to setup Redis cluster.
+
+## How to setup a Redis cluster
+
+Redis requires at least three master nodes for a functional cluster. Read more on Redis cluster tutorial [documentation]([https://redis.io/topics/cluster-tutorial](https://redis.io/topics/cluster-tutorial)) to understand other requirements and limitations of Redis cluster. Redis cluster supports Redis master slave model, but commcare-cloud supports running Redis cluster with master nodes only. To setup Redis cluster using commcare-cloud, follow below steps
```

Comment:
nit: if there is one, should we add the reason why we support only master nodes.